grails-app/services/org/pih/warehouse/report/ReportService.groovy

```diff
@@ -1133,4 +1134,33 @@ class ReportService implements ApplicationContextAware {
 
         return new PaginatedList<InventoryAuditSummary>(data, results.totalCount);
     }
-}
\ No newline at end of file
+
+    Map getTotalCount(IndicatorCommand command) {
+        return [
+                type: "totalCount",
+                number: 1500
+        ]
+    }
+
+    Map getItemsCounted(IndicatorCommand command) {
+        return [
+                type: "itemsCounted",
+                number: 1200
+        ]
+    }
+
+    Map getTargetProgress(IndicatorCommand command) {
+        return [
+                type: "targetProgress",
+                number: 42,
+                subtitleValue: "1281/26182",
+        ]
+    }
+
+    Map getNotFinishedItems(IndicatorCommand command) {
+        return [
+                type: "notFinishedItems",
+                number: 300
+        ]
```

Comment:
I think we can use more appropriate naming of the properties, and a little bit more "generic"
type: sounds like a type of the indicator, but it looks to be the name of that indicator, so I would call it `name`
number: I think it can be `value`, so in every indicator it can be value, not number, string, etc.
subtitleValue: Because we have value, this one can be named `subvalue`